src/slack_block_util.ts

```diff
@@ -306,34 +322,34 @@ export function replaceVoterPanelBlocks(
 // This function mutates the blocks input.
 export function populateDropdownNewInitialValue(
   blocks: SlackBlock[],
+  actionId: SlackActionId,
   newInitialValue: VoterStatus
-): void {
-  const voterStatusOptions = getVoterStatusOptions();
-  const isVoterStatusOption = Object.keys(voterStatusOptions).includes(
-    newInitialValue
-  );
+): boolean {
   for (const i in blocks) {
     const block = blocks[i];
     if (block.type === 'actions') {
       const elements = block.elements;
       for (const j in elements) {
         const element = elements[j];
-        if (isVoterStatusOption) {
+        if (element.action_id === actionId) {
           if (element.type === 'static_select') {
-            element.initial_option.value = newInitialValue;
-            element.initial_option.text.text =
-              voterStatusOptions[newInitialValue];
-            // Javascript modifies the blocks by reference, so end but don't return anything.
-            return;
+            // Assume new options is already in the list of old options
+            element.initial_option = element.options.find(
```

Comment:
If `newInitialValue` is null, will the `initial_option` end up being set to null, and this function would return true?

Reply:
If we don't expect newInitialValue to ever be null, should we have a fallback to leave the `initial_option` as-is and maybe throw an error that logs to Sentry (and have the function return false)?